### PR TITLE
322 check try except block for right logging

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**

### CODE REVIEW

File: proxy/indexer/indexer.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def neon_addr_fmt(neon_tx: NeonTxInfo):
     def _getadd_tx(self, storage_account, neon_tx=None, blocked_accounts=[str]) -> NeonTxObject:
         tx = self.state.get_tx(storage_account)
         if tx and neon_tx and tx.neon_tx and (neon_tx.sign != tx.neon_tx.sign):
-            self._log_warning(f'storage {storage_account}, tx.neon_tx({tx.neon_tx}) != neon_tx({neon_tx})')
+            self.warning(f'storage {storage_account}, tx.neon_tx({tx.neon_tx}) != neon_tx({neon_tx})')
```
Comment (afalaleev) [inline proxy/indexer/indexer.py]: self._log_warning adds information about the current parsed instruction:
https://github.com/neonlabsorg/proxy-model.py/pull/467/files#diff-11de75dcede9b24f514596cfdb04fa1c34c8da38f06ac74f93bfc012076783cbL260

Comment (afalaleev) [reply proxy/indexer/indexer.py]: https://github.com/neonlabsorg/proxy-model.py/pull/467/files#diff-11de75dcede9b24f514596cfdb04fa1c34c8da38f06ac74f93bfc012076783cbL282

{self} -> is replaced with self.__str__()

**Review by afalaleev — CHANGES_REQUESTED** [review]

File: proxy/indexer/indexer.py (hunk 1 above)
Comment (vasiliy-zaznobin) [reply proxy/indexer/indexer.py]: Added current parsed instruction in the log string

**Review by afalaleev — DISMISSED** [review]

File: proxy/indexer/indexer.py [hunk 2]
```diff
@@ -369,7 +366,7 @@ def _init_tx_from_holder(self, holder_account: str, storage_account: str, blocke
 
         rlp_error = tx.neon_tx.decode(rlp_sign=rlp_sign, rlp_data=bytes(rlp_data))
         if rlp_error:
-            self._log_warning(f'Neon tx rlp error "{rlp_error}"')
+            self.error(f'Neon tx rlp error: {rlp_error}, on instruction: {self}')
```
Comment (afalaleev) [inline proxy/indexer/indexer.py]: The format is changed here.

Comment (vasiliy-zaznobin) [reply proxy/indexer/indexer.py]: Fixed

**Review by afalaleev — APPROVED** [review]

**Pull request merged**